### DISCUSSION TITLE
Version on MS Store

### DISCUSSION THREAD

**Discussion by TechStudent10** [ThioJoe/YT-Spammer-Purge#91] [Ideas]
Now that Microsoft is allowing developers to publish Win32 applications to the Microsoft Store, this program could have a version on the MS Store. There is of course one big problem, how do you handle logging in? If the client needs to provide the `client_secrets.json` file, then how might you do this with a version on MS Store? I mean, you could ask the user for the location of the file each time they want to use this program, or you could ask them once and store it somewhere.

Just spitballing.

**Comment by KendallDoesCoding** [comment]
> 

While anyone who's interesting in deleting spam comments will ideally do a search and find this project by ThioJoe, It's something to think about, but that's up to @ThioJoe

**Comment by ThioJoe** [comment]
Microsoft Store has a lot of requirements that would be too much of a pain to fulfill

**Reply by TechStudent10** [reply]
bruh

**Reply by TechStudent10** [reply]
Microsoft allows Win32 apps but 
>  has a lot of requirements that would be too much of a pain to fulfill

But, the Python Foundation was able to release Python 3.9 onto the Microsoft Store, months before Windows 11 and this MS Store change.

Edit: Fixed formatting

**Comment by KendallDoesCoding** [comment]
Yes, and I also think you need to pay a developer free.

Like, many devs face issues. You need to keep to date with the TOS, as they
add new and new rules for app devs without any warnings, nor do they email
any devs saying we changed our rules.

**Reply by TechStudent10** [reply]
A FEE?!

**Reply by KendallDoesCoding** [reply]
Individual accounts cost approximately $19 USD, and company accounts cost
approximately $99 USD (the exact amounts may vary depending on your country
or region
<https://docs.microsoft.com/windows/uwp/publish/account-types-locations-and-fees>).
This is a one-time registration fee and no renewal is required.

For reference:
https://developer.microsoft.com/en-us/microsoft-store/register/

It's in the first question in these FAQ's

**Reply by KendallDoesCoding** [reply]
Google, Apple, Android, everyone has these fees for app devs.

**Reply by KendallDoesCoding** [reply]
But, it's worth paying if you want to be a dev, coz you can earn from people downloading the app too, but as he said, there are a lot of requirements

**Comment by KendallDoesCoding** [comment]
Individual accounts cost approximately $19 USD, and company accounts cost
approximately $99 USD (the exact amounts may vary depending on your country
or region
<https://docs.microsoft.com/windows/uwp/publish/account-types-locations-and-fees>).
This is a one-time registration fee and no renewal is required.

For reference:
https://developer.microsoft.com/en-us/microsoft-store/register/

It's in the first question in these FAQ's

**Reply by TechStudent10** [reply]
BRUH

**Comment by KendallDoesCoding** [comment]
Google, Apple, Android, everyone has these fees for app devs.

**Reply by TechStudent10** [reply]
WHY?!?

**Comment by KendallDoesCoding** [comment]
Idk, its a stupid rule. Ig because, its a platform where you can earn money
from people downloading the app, but there too they take a cut. stupid
rule, but we cant do anything abt it.

**Reply by TechStudent10** [reply]
yeah
